### PR TITLE
Add support to table_name_prefix and table_name_suffix

### PR DESCRIPTION
**Pull request by cavi21**
Address the issue #207 

### What is the purpose of this pull request?
As described in #207, there is no support for when the `config.active_record.table_name_prefix` or `config.active_record.table_name_suffix` are defined. This PR fixes what's relevant to `logidze`.

### What changes did you make? (overview)
 - I add the method `full_table_name` as suggested by @palkan in the issue #207. 
 - Quoted all the references to table or triggers names, in order to safely handle prefixes or suffixes that include for instance a `-` (hyphen). I explore the idea of using the `quote_table_name()` method on the connection for it but was not sure what approach do you prefer? what I thought that maybe we can have something like:

 ```ruby
def quote_db_object_name(object_name)
  ActiveRecord::Base.connection.quote_table_name(object_name)
end
```
The idea of having a method like this, is so that we can pass for instance `logidze_on_#{full_table_name}` and not only table names. And also because this gem supports only PG not sure if make sense to have this in there. 

 - On the `Logidze::Model` I tweak the method `reload_log_data` in order to prevent the raise of the `ActiveRecord::Sanitization::ClassMethods#disallow_raw_sql!`
 - Changes on the specs to properly match the `"` (quotes) on the object names

### Is there anything you'd like reviewers to focus on?

- It's important to have in mind that this changes did not addresses the issue with the naming of the `fx` functions or triggers. I open [an issue](https://github.com/teoljungberg/fx/issues/81) over there in order to know if it's something that's in the scope of the gem 
- Also because there are multiline and onliners SQL sometimes we need a escaped quote and sometimes we don't not sure if you want to move everything to use the `<<~SQL` and in such the quotation can be more consistent.

### Checklist

- [x] I've added tests for this change
- [x] I've added a Changelog entry
- [ ] ~I've updated a documentation (Readme)~

### CODE REVIEW

File: spec/generators/model_generator_spec.rb [hunk 1]
```diff
@@ -9,6 +9,15 @@
   let(:use_fx_args) { USE_FX ? [] : ["--fx"] }
   let(:fx_args) { USE_FX ? ["--no-fx"] : [] }
 
+  let(:table_name_prefix) { TABLE_NAME_PREFIX }
+  let(:table_name_suffix) { TABLE_NAME_SUFFIX }
+
+  let(:full_table_name) {
```
Comment (palkan) [inline spec/generators/model_generator_spec.rb:15]: I think, we can define a plain Ruby method here:

```ruby
def full_table_name(table_name)
  # ...
end
```

**Review by palkan — CHANGES_REQUESTED** [review]
Thanks!

> we don't not sure if you want to move everything to use the <<~SQL and in such the quotation can be more consistent

That's a good idea; let's refactor to use `<<~` everywhere.

P.S. And there are some RuboCop failures. Please, take a look: https://github.com/palkan/logidze/runs/4789659137?check_suite_focus=true

File: spec/generators/model_generator_spec.rb (hunk 1 above)
Comment (cavi21) [reply spec/generators/model_generator_spec.rb:15]: Perfect @palkan I will look into the stuff that you mentioned! thanks for the time of looking into it!

**Comment by cavi21** [comment]
Hi @palkan I just pushed the requested changes, let me know what do you think

**Review by palkan — APPROVED** [review]
Thanks!

**Comment by palkan** [comment]
Merged into master (only relevant commits, w/o merge ones).

**Comment by cavi21** [comment]
Thanks @palkan for all the effort put into this! have a great week!

**Pull request closed without merging**